src/components/Posts/ExpandedPostItem.tsx

```diff
@@ -217,6 +227,13 @@ const ExpandedPostItem = () => {
         </Stack>
       </CardActions>
       <Divider variant="middle" />
+      {openModal && (
```

Comment:
We don't need this conditional render if we're using the `open` prop on the modal. I think we should choose either the parent or the child to control the render, not bot. So we could either set `open` in the child to always be true and keep this conditional render using `openModal`, or drop the conditional render and pass `openModal` to the child

Reply:
okay good eye, I'll remove the conditional render and just pass it down to the child as a prop